pkg/provider/gardener/ruleset/disak8sstig/rules/242400.go

```diff
@@ -141,7 +142,7 @@ func (r *Rule242400) Run(ctx context.Context) (rule.RuleResult, error) {
 
 	allPods, err := kubeutils.GetPods(ctx, r.ClusterClient, "", labels.NewSelector(), 300)
 	if err != nil {
-		checkResults = append(checkResults, rule.ErroredCheckResult(err.Error(), shootTarget.With("kind", "podList")))
+		checkResults = append(checkResults, rule.ErroredCheckResult(err.Error(), shootTarget.With("kind", "PodList")))
 	} else {
 		var pods []corev1.Pod
```

Comment:
```suggestion
		checkResults = append(checkResults, rule.ErroredCheckResult(err.Error(), shootTarget.With("kind", "PodList")))
		return rule.Result(r, checkResults...), nil
	}

	var pods []corev1.Pod
```

Lets remove the nesting here